cpp/src/Ice/DynamicLibrary.cpp

```diff
@@ -19,24 +19,22 @@ using namespace std;
 
 IceInternal::DynamicLibrary::DynamicLibrary() : _hnd(nullptr) {}
 
-IceInternal::DynamicLibrary::~DynamicLibrary()
+IceInternal::DynamicLibrary::~DynamicLibrary() = default;
+/*
```

Comment:
We should get rid of this code block and move the comment above onto the function